mlocal/checks/project-post.chk

```diff
@@ -46,6 +46,11 @@ config_add_def CONTAINER_FINALDIR LOCALSTATEDIR \"/singularity/mnt/final\"
 config_add_def CONTAINER_OVERLAY LOCALSTATEDIR \"/singularity/mnt/overlay\"
 config_add_def SESSIONDIR LOCALSTATEDIR \"/singularity/mnt/session\"
 
+build_runtime=0
+if [ "$host" = "unix" ]; then
```

Comment:
For now we are only building the runtime for `unix` hosts. We could potentially have a `unix` target that doesn't include the `runtime` too, but haven't seen a need for that.